package.json

```diff
@@ -25,9 +25,27 @@
     "babel-plugin-wildcard": "^5.0.0",
     "eslint": "^5.5.0",
     "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-import": "2.12.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.1"
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.0.7",
+    "prettier": "^1.19.1"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix"
+    ],
+    "*.{json,md,scss,yaml,yml}": [
+      "prettier --write"
+    ]
+  },
+  "husky": {
```

Comment:
does it mean that running `yarn` installs the pre-commit hook?

Reply:
Yes, [Husky](https://github.com/typicode/husky) does that.